This disables usage of new Linux 5.2 system calls.

This is necessary as musl has picked up the syscall numbers for these in its
most recent release, but our kernel-libc-headers is still 4.19, which breaks
build.

Glibc is fine because it uses syscall numbers from these kernel headers
instead of defining its own.

--- core-shim.c
+++ core-shim.c
@@ -24,6 +24,19 @@
  */
 #include "stress-ng.h"

+#ifdef __NR_fsopen
+#undef __NR_fsopen
+#endif
+#ifdef __NR_fsmount
+#undef __NR_fsmount
+#endif
+#ifdef __NR_fsconfig
+#undef __NR_fsconfig
+#endif
+#ifdef __NR_move_mount
+#undef __NR_move_mount
+#endif
+
 /*
  *  Various shim abstraction wrappers around systems calls and
  *  GCC helper functions that may not be supported by some
--- stress-ramfs.c
+++ stress-ramfs.c
@@ -24,6 +24,19 @@
  */
 #include "stress-ng.h"
 
+#ifdef __NR_fsopen
+#undef __NR_fsopen
+#endif
+#ifdef __NR_fsmount
+#undef __NR_fsmount
+#endif
+#ifdef __NR_fsconfig
+#undef __NR_fsconfig
+#endif
+#ifdef __NR_move_mount
+#undef __NR_move_mount
+#endif
+
 static const stress_help_t help[] = {
 	{ NULL,	"ramfs N",	 "start N workers exercising ramfs mounts" },
 	{ NULL,	"ramfs-ops N",	 "stop after N bogo ramfs mount operations" },
